packages/common/src/types/index.ts

```diff
@@ -126,9 +126,10 @@ export type CurrentUser = {
 export type CustomTemplate = {
   color?: string;
   title: string;
-  id: string;
+  id?: string;
```

Comment:
I think this one is not optional 🤔, same for `url` and `description` (though `description` is probably `string | null`).